readme.md

```diff
@@ -16,7 +16,7 @@ The library also offers synthetic streams (pointcloud, depth aligned to color an
 
 The library aims to provide an easy to use API and tools for computer vision professionals, game developers and other **Intel® RealSense™** technology enthusiasts.
 
-Developer kits containing the necessary hardware to use this library are available for purchase at [realsense.intel.com/buy](realsense.intel.com/buy).
+Developer kits containing the necessary hardware to use this library are available for purchase at [https://realsense.intel.com/buy](https://realsense.intel.com/buy).
```

Comment:
I think it would be better to leave the link's text as it was and just change the link itself